qiskit_algorithms/__init__.py

```diff
@@ -11,29 +11,43 @@
 # that they have been altered from the originals.
 
 """
-=====================================
-Algorithms (:mod:`qiskit_algorithms`)
-=====================================
-It contains a collection of quantum algorithms, for use with quantum computers, to
-carry out research and investigate how to solve problems in different domains on
-near-term quantum devices with short depth circuits.
-
-Algorithms configuration includes the use of :mod:`~qiskit_algorithms.optimizers` which
-were designed to be swappable sub-parts of an algorithm. Any component and may be exchanged for
-a different implementation of the same component type in order to potentially alter the behavior
-and outcome of the algorithm.
+============================================
+Qiskit Algorithms (:mod:`qiskit_algorithms`)
+============================================
+Qiskit Algorithms is a library of quantum algorithms for quantum computing with
+`Qiskit <https://qiskit.org>`_.
+These algorithms can be used to carry out research and investigate how to solve
+problems in different domains on simulators and near-term real quantum devices
+using short depth circuits.
+
+The library includes some algorithms, for example the :class:`.NumPyMinimumEigensolver`, which take
+the same input as their quantum counterpart but solve the problem classically. This has utility in
+the near-term, where problems are still tractable classically, to validate and/or act as a reference.
+There are also classical :mod:`.optimizers` for use with variational algorithms such as :class:`.VQE`.
+
+Other common subroutines used by this module's algorithms are :mod:`.gradients` and
+:mod:`.state_fidelities`. Any of these may also be used directly if you have need of such function.
+
+The quantum algorithms here all use ``Primitives`` to execute quantum circuits. This can be an
```

Comment:
How about adding a link to the primitives docs?
```suggestion
The quantum algorithms here all use 
`Primitives <https://qiskit.org/documentation/apidoc/primitives.html>`__ 
to execute quantum circuits. This can be an
```

Reply:
We can. I thought about this when I did the text and ended up not adding it. Why - I do not find that link all that helpful if you are coming to primitives not know anything and need the link. I thought about linking the medium blog but that did not do it for me. I thought about both plus links to the Aer and Runtime - that seemed a bit much. And Qiskit has some howtos that had been done but I think they are yet to be published. In the end I left it with no links. But it can have a link here - maybe that content will be improved or maybe we can have a better link going forwards. But I guess its fine for now.